stdlib/3/os/__init__.pyi

```diff
@@ -375,6 +531,16 @@ else:
     def walk(top: AnyStr, topdown: bool = ..., onerror: Optional[Callable[[OSError], Any]] = ...,
              followlinks: bool = ...) -> Iterator[Tuple[AnyStr, List[AnyStr],
                                                         List[AnyStr]]]: ...
+if sys.version_info >= (3, 3):
+    def fwalk(top: AnyStr = ..., topdown: bool = ...,
+              onerror: Optional[Callable] = ..., *, follow_symlinks: bool = ...,
+              dir_fd: Optional[int] = ...) -> Iterator[Tuple[AnyStr, List[AnyStr],
+                                             List[AnyStr], int]]: ...  # Unix only
+    def getxattr(path: _FdOrPathType, attribute: AnyStr, *, follow_symlinks: bool = ...) -> bytes: ...  # Linux only
```

Comment:
AnyStr generally doesn't make sense if it's only used once, since it's meant to express that multiple values should be the same type (either str or bytes). Also, the argument clinic types suggest these "attributes" are paths, so maybe PathLike should also be allowed.

Reply:
Yeah... sorry. Interesting thing is that the documentation says that `fwalk()` "behaves exactly like `walk()`", but that's not the case. If `walk()` is called with a *bytes* path, the results are also *bytes*, hence the use of *AnyStr*. However, `fwalk()` returns strings regardless of the input path type.